packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -462,26 +493,148 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Some popups, like iOS's volume control popup, choose to render a blurred
   /// area without any white paint covering it. To achieve this effect,
   /// [isSurfacePainted] should be set to false.
+  ///
+  /// Defaults to true.
   final bool isSurfacePainted;
 
   /// The widget below this widget in the tree.
-  final Widget? child;
+  ///
+  /// Because [CupertinoPopupSurface] is composed of proxy boxes, which mimic
+  /// the size of their child, a [child] is required to ensure that this surface
+  /// has a size.
+  final Widget child;
+
+  /// The default strength of the blur applied to widgets underlying a
+  /// [CupertinoPopupSurface].
+  ///
+  /// Setting this to 0 will remove the blur filter.
```

Comment:
These two lines don't apply to this variable (a static value), but the field. Move them to `blurSigma`.

Reply:
Done! Mistake on my end.